.cargo/config

```diff
@@ -22,5 +22,8 @@ rustflags = [
   "-C", "link-arg=-Tlink.x",
 ]
 
+[target.armv7-unknown-linux-gnueabihf]
+linker = "arm-linux-gnueabihf-gcc"
+
```

Comment:
Is this really required? I don't have this package installed on my M1 mac and the rpi example compiled with no errors.